README.md

```diff
@@ -176,15 +175,44 @@ You have to access it using:
 MicrocksContainer microcks = ensemble.MicrocksContainer;
 microcks.ImportAsMainArtifact(...);
 ```
+
+To activate async features (WebSocket), you can use `WithAsyncFeature()` method.
+
+```csharp
+MicrocksContainersEnsemble ensemble = new MicrocksContainerEnsemble(network, MicrocksImage)
+    .WithMainArtifacts("pastry-orders-asyncapi.yml")
+    .WithAsyncFeature();
+
+ensemble.StartAsync();
+```
+
+#### Asynchronous API support
+Asynchronous API feature needs to be explicitly enabled as well.
+In case you want to use it for mocking purposes,
+you'll have to specify additional connection details to the broker of your choice.
+
+To add a note indicating that it is not necessary to call `WithAsyncFeature()` when an additional method exists.
```

Comment:
We still have to call `withAsyncFeature()` when specifying a Kafka connection on the other languages. But it could be a nice enhancement!